web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -132,12 +121,36 @@ export class LayerDialogComponent implements OnDestroy {
     });
   }
 
-  onProjectLoaded(project: Project) {
-    if (this.layerId === ':new') {
+  initProject(
```

Comment:
This method doesn't actually initialize the project, it initializes the LayerDialogComponent, so `init()` or similar would be more appropriate here. But really, this is just part of the constructor logic logic and can be inlined there if you like.